docs/libs/ml/svm.md

```diff
@@ -186,5 +199,5 @@ svm.fit(trainingDS)
 val testingDS: DataSet[Vector] = env.readVectorFile(pathToTestingFile)
 
 // Calculate the predictions for the testing data set
-val predictionDS: DataSet[LabeledVector] = svm.predict(testingDS)
+val predictionDS: DataSet[(FlinkVector, Double)] = svm.predict(testingDS)
```

Comment:
Maybe if I add a comment clarifying what I'm doing? My problems is that Scala Vector is imported by default and if you don't import the FlinkML Vector it will lead to problems.